runtime-attach/build.gradle.kts

```diff
@@ -11,4 +11,10 @@ dependencies {
 
   // Used by byte-buddy but not brought in as a transitive dependency.
   compileOnly("com.google.code.findbugs:annotations")
+
+  testImplementation("io.opentelemetry.javaagent:opentelemetry-javaagent:1.13.1")
```

Comment:
Can you add `io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha` to `dependencyManagement`? This way you won't have to pass the version here.